src/components/Button/Button.Props.ts

```diff
@@ -78,3 +90,13 @@ export enum ButtonType {
   command,
   icon
 }
+
+export enum IconColor {
+  neutralSecondary,
+  white
```

Comment:
This seems like a really bad idea. why would it not just have some default in css, and then the iconColor becomes a string, that if provided overrides it with an inline style?

We can't be hardcoding "neutralSecondary" and "white" in enums.